### DISCUSSION TITLE
Three phase battery data query using API

### DISCUSSION THREAD

**Discussion by vmotakat** [GRIDAPPSD/gridappsd-forum#132] [API] [answered]
PREFIX r: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
PREFIX c: <http://iec.ch/TC57/CIM100#>
PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
SELECT ?inverter_mrid ?meas_mrid ?phase
WHERE {
  VALUES ?feeder_mrid {"_8F9EA8EC-1CC5-4ABC-8287-3A6D363DB962"}
  ?s r:type ?type.
  ?s c:IdentifiedObject.mRID ?meas_mrid.
  ?s c:Measurement.PowerSystemResource ?eq.
  ?s c:Measurement.Terminal ?trm.
  ?s c:Measurement.phases ?phsraw.
  {bind(strafter(str(?phsraw),"PhaseCode.") as ?phase)} .
  ?eq c:IdentifiedObject.mRID ?inverter_mrid.
  ?eq r:type c:PowerElectronicsConnection.
  ?eq c:Equipment.EquipmentContainer ?fdr.
  ?fdr c:IdentifiedObject.mRID ?feeder_mrid.
}
ORDER BY ?inverter_mrid

I have used above query in blaze graph. Test feeder having three phase batteries along with three phase PV models. I am getting none under phase section  as shown in the bellow image

 
<img width="797" alt="Screen Shot 2022-06-22 at 3 47 10 PM" src="https://user-images.githubusercontent.com/102381298/175124259-8bc11a93-780a-471d-8f4c-3386c77684e6.png">

Can any one help me to understand why multiple times same object MRID and measurement MRID for all phases A, B, C  all objects.
@poorva1209

**Comment by AAndersn** [comment, marked as answer]
This was answered in #133
